cdap-ui/app/features/adapters/controllers/tabs/runs/tabs/status-ctrl.js

```diff
@@ -57,4 +59,61 @@ angular.module(PKG.name + '.feature.adapters')
 
 
       });
+
+    var context = 'namespace.' + $state.params.namespace +
```

Comment:
On second thought, it seems like you're using the string `context` on line 71 below, so maybe ignore the suggestion in previous comment.